keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/VerificationEvaluator.kt

```diff
@@ -0,0 +1,12 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.verification.VerificationStatus
+
+interface VerificationEvaluator<VERIFICATION: Verification> {
+  val supportedVerification: Pair<String, Class<out Verification>>
+
+  fun evaluate() : VerificationStatus
+
+  fun start(verification: Verification)
```

Comment:
This should be using the generic type, right?
```suggestion
  fun start(verification: VERIFICATION)
```

Reply:
Hmm, it's actually annoying because then you'd have to make a cast at the call-site

Reply:
Ah, right. I guess just in the `supportedVerification` field above, then. I just noticed we were not using the generic type anywhere in the interface. 🙂